go.mod

```diff
@@ -168,9 +168,10 @@ require (
 	golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1
 	gopkg.in/cheggaaa/pb.v1 v1.0.28
 	gotest.tools v2.2.0+incompatible
-	lukechampine.com/blake3 v1.1.7 // indirect
 )
 
 replace github.com/filecoin-project/filecoin-ffi => ./extern/filecoin-ffi
 
 replace github.com/filecoin-project/test-vectors => ./extern/test-vectors
+
+replace github.com/ipfs/go-path => github.com/ipfs/go-path v0.0.7
```

Comment:
Is this needed?

Reply:
Looks like this is needed otherwise node chain API implementation would need an upgrade. 

Resolving. Please unresolve if this is a blocker.

Reply:
Yeah, we shouldn't have this here unless it means a big change. I can be convinced to keep this if we'll fix this soon in a followup PR

Reply:
I have upgraded the code here: https://github.com/filecoin-project/lotus/pull/7313/commits/1bb65189af87631b2adc32ccc728f1af1ffee554

Please review carefully as I am not well rehearsed with lotus codebase. 
Resolving.
Thanks